src/sentry/runner/commands/presenters/webhookpresenter.py

```diff
@@ -125,9 +127,18 @@ def invalid_type(
 
     def _send_to_webhook(self, json_data: dict[str, Any]) -> None:
         if settings.OPTIONS_AUTOMATOR_SLACK_WEBHOOK_URL:
-            headers = {"Content-Type": "application/json"}
+
+            json_data_str = json.dumps(json_data)
+
+            secret_key = settings.OPTIONS_AUTOMATOR_HMAC_SECRET.encode()
+            signature = hmac.new(secret_key, json_data_str.encode(), hashlib.sha256).hexdigest()
+
+            headers = {
+                "Content-Type": "application/json",
+                "X-Hub-Signature-256": f"sha256={signature}",
```

Comment:
Verify this is the correct header name. `X-Hub-Signature-256` is what GitHub uses for its webhooks. I suspect we might use something different for options-automator? 🙂